dapr/proto/common/v1/common.proto

```diff
@@ -95,11 +94,11 @@ message StateItem {
   // The metadata which will be passed to state store component.
   map<string,string> metadata = 4;
 
-  // Options for concurrency, consistency, and retry_policy to save the state.
+  // Options for concurrency, and consistency to save the state.
```

Comment:
nit, there is extra comma between concurrency & and

Reply:
fixed it now. thanks.